docs/contribute.md

```diff
@@ -19,12 +19,35 @@ We ask that before significant effort is put into code changes, that we have agr
 
 An ADR is an Architectural Decision Record.  This allows consensus on the direction forward and also serves as a record of the change and motivation.  [Read more here](adrs/README.md)
 
-## Development Life Cycle
-
-### Required Dev Dependencies
+## Required Dev Dependencies
 
 ![Win](res/win_sm.png) ![*nix](res/linux_sm.png)  Git for Windows and Linux [Install Here](https://git-scm.com/downloads) (needed for dev sh script)
 
+## Quickstart: Run a job from a real repository
```

Comment:
I wrote up a quickstart session as a good way to get to know the repo, let me know what you think.